### ISSUE TITLE
ExportOptions.plist file is not created by default.

### ISSUE THREAD

**Issue by ejebleed** [fastlane/fastlane#5907]
Documentation claims:
Export options

Since Xcode 7, gym is using new Xcode API which allows us to specify export options using plist file. By default gym creates this file for you and you are able to modify some parameters by using export_method, export_team_id, include_symbols or include_bitcode. 

it is not true.

i'm using this command line to build the app:
`gym --clean --scheme ${XCODE_PROJECT_TARGET_NAME} --configuration Release --include_bitcode false --archive_path ${BUILDS_DIR}${PACKAGE_NAME}.xcarchive --output_directory $BUILDS_DIR --output_name "${PACKAGE_NAME}.ipa" --xcargs PROVISIONING_PROFILE=$APP_PROVISION_UUID`

Using gym installed by **gem install gym**

gym version **1.7.0**
ruby 2.0.0p648 (2015-12-16 revision 53162) [universal.x86_64-darwin15]
no ruby environment manager

**Comment by TKBurner** [comment]
Hi @ejebleed Thanks for reaching out. Can you run `sudo gem install fastlane` and then share your full input and output with me? I want to take a look. Thanks!

**Comment by ejebleed** [comment]
Hi @TKBurner 
I'm not using the whole fastlane package. I'm using gym only.  Is it safe to install whole fastlane on top of it? I must be sure it will not break any part of my environment here.

I dont know if it matters but that gym command is executed within shell script. So if I want to use it manually then I open terminal (Mac), type sh my_script.sh and it executes that long gym command I stated above.

**Comment by KrauseFx** [comment]
@ejebleed Thanks for the report

You said it's not true, how come? What part is not working?

**Comment by ejebleed** [comment]
@KrauseFx , it is very possible that I misunderstood the documentation, but as documentation says:

`Since Xcode 7, gym is using new Xcode API which allows us to specify export options using plist file.` **By default gym creates this file for you** `and ...`

So I assumed that it would generate the _ExportOptions.plist_ file somewhere. But looks like it did not. At least the directory where the _gym_ command is executed did not contain any generated files after executing _gym_ command.

Btw, personally I do not have any trouble with this specific issue. But in search of the solution for one another bug, I came through trying to use the ExportOptions.plist file as one possible solution to that bug. And there it was where I noticed that the file was not generated by default as documentation claimed.

**Comment by TKBurner** [comment]
Hi @ejebleed Thanks for reaching out on this. Are you able to specify a export_options plist using that parameter? Thanks!

**Comment by KrauseFx** [comment]
It's generated in a temporary directory, it's all handled for you. 

Did you run into any issue while building, or is it just the plist file you're looking for? If you run `gym --verbose`, you'll see the exact commands, including the path to the plist file

**Comment by ejebleed** [comment]
No, I did not run into any issue here. I just noticed that there are no ExportOptions.plist file. I was expecting it to appear in my source code directory where the gym command was executed. Wanted to take it as a sample, modify it and use it again, But now I know that it can be found somewhere in temporary directory. So no problem here. :)
Maybe, documentation could be more specific on file locations by stating somehow where one or the other file can be found if needed.
